pkg/render/manager.go

```diff
@@ -292,6 +292,10 @@ func (c *managerComponent) managerDeployment() *appsv1.Deployment {
 		initContainers = append(initContainers, c.cfg.VoltronLinseedKeyPair.InitContainer(ManagerNamespace))
 	}
 
+	managerPodContainers := []corev1.Container{c.managerEsProxyContainer(), c.voltronContainer()}
+	if !c.cfg.Tenant.SingleTenant() && !c.cfg.Tenant.MultiTenant() {
```

Comment:
Can we simplify this check to be `if c.cfg.Tenant != nil`. I think we only need to install this for zero tenants, aka Enterprise installations.

Reply:
Sure yeah could do.